benchmark/src/DriverBench/ParallelMultiFileImportBench.php

```diff
@@ -73,20 +71,6 @@ public function beforeIteration(): void
         $database->createCollection(Utils::getCollectionName());
     }
 
-    /**
-     * Using Driver's BulkWrite in a single thread.
-     * The number of files to import in each iteration is controlled by the "chunk" parameter.
-     *
-     * @param array{chunk:int} $params
-     */
-    #[ParamProviders(['provideChunkParams'])]
-    public function benchBulkWrite(array $params): void
```

Comment:
This subject essentially tests the same as `benchInsertMany`, so I decided to skip it in order to shave ~6 minutes off the run time.

Reply:
We could keep this and remove the other that is a little less efficient

Reply:
The reason I removed `bulkWrite` and kept `insertMany` is that benchmarking the latter also exposes performance regressions introduced in the library, while the other only tests the extension.